CMakeLists.txt

```diff
@@ -38,3 +38,5 @@ add_executable(tmatrix ${tmatrix_SRC})
 include_directories(include)
 
 install(TARGETS tmatrix DESTINATION usr/bin)
+install(FILES tmatrix.6 DESTINATION usr/share/man/man6)
```

Comment:
Man pages are usually in a gzip-ed form in man directory so it isn't a simple as this. It's not hard to do with a shell script (see install.sh in [installation.tar.gz](https://github.com/M4444/TMatrix/releases)) but it's a bit tricky with CMake. There is an inbuilt tar command which you can find [here](https://cmake.org/cmake/help/v3.14/manual/cmake.1.html#run-a-command-line-tool) but I just couldn't get it to create a proper gzip. Maybe you can figure it out or find another way, like calling an external command.

Reply:
Yes, I forgot about this since I was thinking about the Arch package. The Arch `makepkg` command automatically compresses any uncompressed man pages before creating the package. I have added steps to `CMakeLists.txt` to produce a compressed man page, but I'm not so sure that CMake is intended to be used this way :p You can choose if you want to go with the CMake method, or if you prefer to do it in an install script and leave it to the user/packager when building manually.

Reply:
> I'm not so sure that CMake is intended to be used this way :p

Who knows how CMake was intended to be used but I do like this way, I just couln't be bothered to figure out so far.
I would like to have the compression and the install() command for the man page all in one commit. Can you put these changes together and make them the first commit? It's easier to review than the completion script so it will be committed first.
Also add a period at the end of the comment in the CMakeLists.txt.

Reply:
Done.